### PR TITLE
Add baseline assistant functionality

### PR DESCRIPTION
**Pull request by IhorNi**

### CODE REVIEW

File: contacts24/models/record.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from datetime import date, datetime
 from typing import List, Optional
 
-from errors import InaccurateBirthdayFormat, InaccuratePhoneFormat
+from contacts24.errors import InaccurateBirthdayFormat, InaccuratePhoneFormat
 
 DATE_FORMAT = "%d.%m.%Y"
```
Comment (SavchukR) [inline contacts24/models/record.py]: @IhorNi  Date format is already present in `config.py`

File: contacts24/contacts.py [hunk 2]
```diff
@@ -1,24 +1,118 @@
...
+        return cmd, None
 
-def delete_contact(name: str):
-    pass
+    return cmd, *args
+
+
+def load_contacts_book() -> AddressBook:
+    try:
+        contacts = AddressBook.load_from_file("address_book.json")
```
Comment (SavchukR) [inline contacts24/contacts.py:36]: @IhorNi  could we refer here to the imported constant from `config.py`?

**Review by SavchukR — CHANGES_REQUESTED** [review]

File: contacts24/models/address_book.py [hunk 3]
```diff
@@ -3,12 +3,10 @@
...
 
-from models.record import DATE_FORMAT, Record
-
-ADDRESS_BOOK_FILENAME = "address_book.json"
-WEEKDAYS = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday", "Next Monday"]
-MONDAY_INDEX = 0
-NEXT_MONDAY_INDEX = -1
+from contacts24.models.record import Record
+from contacts24.config import (
+    CONTACTS_FILE, WEEKDAYS, MONDAY_INDEX, NEXT_MONDAY_INDEX, DATE_FORMAT
```
Comment (SavchukR) [inline contacts24/models/address_book.py]: CONTACTS_FILE is now `ADDRESSBOOK_FILE`.
Please merge from `main`

**Review by SavchukR — CHANGES_REQUESTED** [review]
@IhorNi 1 rename of `ADDRESSBOOK_FILE` (see comment).

**Review by SavchukR — APPROVED** [review]

**Review by krisdzhus — COMMENTED** [review]
LGTM

**Pull request merged**